types/memory-cache/index.d.ts

```diff
@@ -1,20 +1,37 @@
 // Type definitions for memory-cache
```

Comment:
Should have `major.minor` at the end, e.g. `// Type definitions for memory-cache 0.2`.